pyworkflow/__init__.py

```diff
@@ -37,7 +37,7 @@
 # if plugins are still compatible, so future hot fixes releases or even micros
 # or minor release should not change this CORE_VERSION. Only, when a new release
 # will break existing plugins, this number needs to be incremented.
-CORE_VERSION = '2.0'
+CORE_VERSION = '3.0'
```

Comment:
Can we use '3.0.0' to keep the same numbering format? Although I know '3.0' is equivalent to '3.0.0' in semantic versioning

Reply:
Ok